docs/examples/grafana/README.md

```diff
@@ -0,0 +1,211 @@
+
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    volumes:
+      - "./dependencies/jaeger/jaeger-ui.json:/etc/jaeger/jaeger-ui.json"
+    command: --query.ui-config /etc/jaeger/jaeger-ui.json
+    environment:
+      - METRICS_STORAGE_TYPE=prometheus
+      - PROMETHEUS_SERVER_URL=http://prometheus:9090
+    ports:
+      - "14250:14250"
+      - "16685:16685" # GRPC
+      - "16686:16686" # UI
+    networks:
+      - static-network
+
+  prometheus:
+    image: prom/prometheus:latest
+    volumes:
+      - "./dependencies/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml"
+    ports:
+      - "9090:9090"
+    networks:
+      - static-network
+
+  grafana:
+    image: grafana/grafana-oss
+    restart: unless-stopped
+    ports:
+      - "3000:3000"
+    networks:
+      - static-network
+
+networks:
+  static-network:
+```
+
+### Application example
+
+Example service mocking a call to a remote API: here the remote API returns apples or bananas.
+We're using the metrics to measure the apple/banana ratio returned by the API,
+and the traces to measure the latency of this API. 
+
+```scala 3
+import cats.effect.Async
+import cats.effect.std.Random
+import cats.implicits.{catsSyntaxApply, toFlatMapOps, toFunctorOps}
+import io.circe.derivation.{Configuration, ConfiguredCodec}
+import org.typelevel.otel4s.{Attribute, AttributeKey}
+import org.typelevel.otel4s.metrics.Meter
+import org.typelevel.otel4s.trace.Tracer
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.FiniteDuration
+
+given Configuration = Configuration.default
+case class ApiData(result: String) derives ConfiguredCodec
+
+trait ExampleService[F[_]] {
+  def getDataFromSomeAPI: F[ApiData]
+}
+
+object ExampleService {
+  def apply[F[_]: Async: Tracer: Meter: Random](
+      minLatency: Int,
+      maxLatency: Int,
+      bananaPercentage: Int
+  ): F[ExampleService[F]] = {
+    val metricsProvider = summon[Meter[F]]
+    metricsProvider
+      .counter("RemoteApi.fruit.count")
+      .withDescription("Number of fruits returned by the API.")
+      .create
+      .map { remoteApiFruitCount =>
+        new ExampleService[F] {
+          private val spanBuilder = Tracer[F].spanBuilder("remoteAPI.com/fruit").build
+
+          override def getDataFromSomeAPI: F[ApiData] = for {
+            latency <- Random[F].betweenInt(minLatency, maxLatency)
+            isBanana <- Random[F].betweenInt(0, 100).map(_ <= bananaPercentage)
+            duration = FiniteDuration(latency, TimeUnit.MILLISECONDS)
+            fruit <- spanBuilder.surround(
+              Async[F].sleep(duration) *>
+                Async[F].pure(if isBanana then "banana" else "apple")
+            )
+            _ <- remoteApiFruitCount.inc(Attribute(AttributeKey.string("fruit"), fruit))
+          } yield ApiData(s"Api returned a $fruit !")
+        }
+      }
+  }
+}
+```
+
+### Run the application
+
+@:select(build-tool)
+
+@:choice(sbt)
+
+```shell
+$ sbt run
```

Comment:
I don't see an entry point in the example above, so will `sbt run` work?

Reply:
Just tried it on my example project, works fine 👌 
On a second thought though I wonder if the scala-cli snippet makes a lot of sense here, it could maybe even be somewhat misleading as I chose `grafana.scala` kind of as a placeholder name.
Should I change/remove it ?